test/integration/metrics_test.go

```diff
@@ -246,7 +246,7 @@ func TestExitedContainersWithoutTTL(t *testing.T) {
 		"/",
 		raw.NewCgroupV1PathParser(),
 		raw.NewPosixSystemCPUReader(),
-		raw.NetDevNetworkStatsGetter{},
+		raw.NewNetDevNetworkStatsGetter(),
```

Comment:
This is out of the scope of this PR but I think we could use some kind of helper to build the testing fetcher (we are repeating the `raw.NewCgroupsV1Fetcher(...` call a lot.

Reply:
Yes it's true